src/pages/resultsView/network/files/mutationMapper.js

```diff
@@ -1115,12 +1115,18 @@ var MutationDetailsTableFormatter = (function()
 		var style = "";
 		var tipStyle = "";
 
-		if (frequency)
+		if (frequency > 0.01)
 		{
 			style = "mutation_table_allele_freq";
 			text = frequency.toFixed(2);
 			tipStyle = tipClass;
 		}
+		else if (frequency < 0.01) 
+		{
+			style = "mutation_table_allele_freq";
+			text = frequency.toFixed(4);
+			tipStyle = tipClass;
+		}
 
```

Comment:
pull the `style = ` and the `tipStyle = ` out of the `if` statement, since they're done the same way no matter what

Reply:
please do this

Reply:
YEah sure.

Reply:
Check now @adamabeshouse

Reply:
now you're not setting them at all. it should be `style="mutation_table_allele_freq" and tipStyle=tipClass above there.